app.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+
+const app = express();
+
+const port = process.env.PORT || 3000;
+
+app.listen(port, console.log(`Server is listning on port ${port} `));
```

Comment:
Hi @tsheporamantso 
You have misspelt listening, kindly correct the spelling! 🤣

Reply:
Hello Reviewer,
Change done as requested 🤣 Thank you!